src/test/java/Homework16.java

```diff
@@ -0,0 +1,20 @@
+public class Homework16 extends BaseTest {
+    @Test
+    public void registratioNavigatio() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.cssSelector("[href="registration"]"));
```

Comment:
Hello,
Your homework 16 has been reviewed, but the test script execute failed. Please recheck and correct it.
Best wishes from the HW team (Khanh, khanh@testpro.io)

![image](https://github.com/irenefox8/A51-QA-Automation-Framework/assets/144403954/60c7fad8-8c7e-4680-b1f3-cc57b941c802)

//HINT: 
1. importing librarys in Java (Java comes with a core set of libraries, including those that define commonly used data types and related behavior
=>Currently, you are missing the import library

```
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.testng.Assert;
import org.testng.annotations.Test;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.chrome.ChromeOptions;
import org.openqa.selenium.WebDriver;
import java.time.Duration;
```

2. At line declare WebElement registrationLink error when use duplicate double quote character
HINT: `WebElement registrationLink = driver.findElement(By.cssSelector("[href='registration']"));`